src/Sentry.AspNetCore/SentryMiddleware.cs

```diff
@@ -162,6 +165,17 @@ private void SyncOptionsScope(IHub newHub)
 
     internal void PopulateScope(HttpContext context, Scope scope)
     {
+        if (_eventProcessors != null)
+        {
+            var toAdd = _eventProcessors.Except(scope.GetAllEventProcessors());
+            scope.AddEventProcessors(toAdd);
+        }
+        if (_eventExceptionProcessors != null)
+        {
+            var toAdd = _eventExceptionProcessors.Except(scope.GetAllExceptionProcessors());
+            scope.AddExceptionProcessors(toAdd);
+        }
+
```

Comment:
The null checks are not necessary.

```suggestion
        scope.AddEventProcessors(_eventProcessors.Except(scope.GetAllEventProcessors()));
        scope.AddExceptionProcessors(_eventExceptionProcessors.Except(scope.GetAllExceptionProcessors()));
```